btf/btf_test.go

```diff
@@ -285,6 +285,19 @@ func TestLoadSpecFromElf(t *testing.T) {
 	})
 }
 
+func TestVerifierError(t *testing.T) {
+	_, err := NewHandle(&Spec{})
```

Comment:
What does an empty Spec represent though? Is it just type Void? Or something else?

Reply:
Not sure what to reply except the obvious: an empty type collection. On the wire, this results in a btf header, no types, and an empty string table, and this will result in a BTF verifier error saying 'No data'.